### PR TITLE
3.7 release

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Review by sannya-singal — COMMENTED** [review]
Thanks for adding the changes for DMS 🚀 Just a minor comment for supported use cases for serverless.

**Review by Pive01 — APPROVED** [review]
LGTM!

File: content/2024-08-23-localstack-release-v-3-7-0/index.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+### Support for DMS Serverless (Enterprise)
+
+LocalStack now supports AWS Database Migration Service (DMS) Serverless. DMS serverless can be used for the sources and targets that LocalStack already supports, and that are also supported by AWS. To simulate the different states that a replication config experiences during provisioning, you can set the `DMS_SERVERLESS_STATUS_CHANGE_WAITING_TIME` environment variable. This setting delays each state change for the configured number of seconds until the replication is actively running. Refer to the official documentation for details on the different states.
+
+Note that on AWS, replication table statistics are automatically deleted after the replication finishes and the configuration is deprovisioned. For parity, the same applies in LocalStack. To delay this deprovisioning process, you can set the `DMS_SERVERLESS_DEPROVISIONING_DELAY` environment variable, which defaults to 60 seconds.
+
+Learn more about DMS in the [LocalStack documentation](https://docs.localstack.cloud/user-guide/aws/dms).
+
+### Chaos Engineering dashboard now uses Chaos API (Enterprise)
```
Comment (webdev51) [inline content/2024-08-23-localstack-release-v-3-7-0/index.md:62]: It says `Enterprise` directly which gives me a feeling that all of this feature is for enterprise users but DynamoDB Error and Kinesis Error experiments are available for Starter tier as well

**Review by sannya-singal — APPROVED** [review]

**Review by webdev51 — APPROVED** [review]
Looks good to me except for the below requested change

**Review by giograno — APPROVED** [review]
Great job from my side 💯

File: content/2024-08-23-localstack-release-v-3-7-0/index.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+However, it's important to note that LocalStack does not support the actual encryption and decryption of objects using SSE-C. Learn more about S3 in the [LocalStack documentation](https://docs.localstack.cloud/user-guide/aws/s3).
+
+### New `EKS_K8S_PROVIDER` environment variable (Pro)
+
+A new configuration variable `EKS_K8S_PROVIDER` has been introduced with two options:
+
+- `k3s` (**default**, using k3d) 
+- `local` (using a mounted kubeconfig). 
+
+The kubeconfig at `/root/.kube/config` no longer influences provider selection, but a warning is issued if it's missing when using the `local` option. Kubeconfig files located at `~/.kube/config` are not automatically mounted; users must manually mount them if necessary.
```
Comment (dfangl) [inline content/2024-08-23-localstack-release-v-3-7-0/index.md:152]:
```suggestion
The kubeconfig inside the LocalStack container at `/root/.kube/config` no longer influences provider selection, but a warning is issued if it's missing when using the `local` option. Kubeconfig files located on the host at `~/.kube/config` are not automatically mounted; users must manually mount them if necessary.
```

Could we maybe clarify it (maybe like this?) if we are talking about paths inside the container (the first) or on the host (the latter).

**Review by dfangl — APPROVED** [review]
LGTM! Just one suggestion for clarifying paths in the container / on the host.

**Review by tinyg210 — APPROVED** [review]

**Review by viren-nadkarni — APPROVED** [review]
SES and EC2 sections look good :100:

File: content/2024-08-23-localstack-release-v-3-7-0/index.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+- [New enhancements in the CloudFormation provider](#new-enhancements-in-the-cloudformation-provider)
+- [New template option for the LocalStack Extensions CLI](#new-template-option-for-the-localstack-extensions-cli-pro)
+- [New enhancements in the EC2 Libvirt VM manager](#new-enhancements-in-the-ec2-libvirt-vm-manager-pro)
+- [New EC2 Kubernetes executor](#new-ec2-kubernetes-executor-enterprise-preview)
+- [New enhancements in the SES provider](#new-enhancements-in-the-ses-provider)
+- [Miscellaneous](#miscellaneous)
+
+### New Lambda Debug Mode
+
+WIP
```
Comment (cloutierMat) [inline content/2024-08-23-localstack-release-v-3-7-0/index.md:58]: Leaving a note here to make sure we don't forget this one! :wink:

**Review by cloutierMat — APPROVED** [review]
That all looks goog to me. Thanks @HarshCasper for preparing these.

I only left a comment to make sure we don't let this one slip! :smile:

**Review by remotesynth — APPROVED** [review]

**Review by lukqw — APPROVED** [review]
👌

**Review by MEPalma — APPROVED** [review]
All good from my side, thanks!

**Pull request merged**